Source/WTF/WTF.xcodeproj/project.pbxproj

```diff
@@ -4591,13 +4597,18 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5D247B7314689C4700E78B76 /* WTF.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 6.0;
```

Comment:
This needs to be moved to WTF.xcconfig (and language version and optimization level may already be defined?).